xds/src/main/java/io/grpc/xds/Bootstrapper.java

```diff
@@ -60,6 +58,15 @@ static Bootstrapper getInstance() {
     return DEFAULT_INSTANCE;
   }
 
+  /** Gets the current Node from the bootstrap file. */
+  public static Node getNode() {
```

Comment:
You don't need to add this. All you need is `Bootstrapper.getInstance().getNode()`. Please see [`BootstrapInfo`](https://github.com/grpc/grpc-java/blob/17c3e48240aef6cecfdfceea5b6ac64c7e34f812/xds/src/main/java/io/grpc/xds/Bootstrapper.java#L207).

Reply:
`getNode()` is what I added to `Bootstrapper` class. 

You probably meant `Bootstrapper.getInstance().readBootstrap().getNode()` which requires making 5 things public: `Bootstrapper`, `Bootstrapper.getInstance()`, `Bootstrapper.readBootstrap()`, `BootstrapInfo` and `BootstrapInfo.getNode()` just to get the `Node` from the Bootstrapper. I think this is better.

Reply:
Yes, `Bootstrapper.getInstance().readBootstrap().getNode()` is what I mean. Yes, you would make everything in `Bootstrapper` and `BootstrapInfo` public, but that's totally fine. `BootstrapInfo.getNode()` is really re-implementing something already have, that's redundant and inelegant.

Reply:
I strongly request to make this change.

`Bootstrapper` is the client to read a system bootstrap file and `BootstrapInfo` is the result (data class) for doing that operation. Adding `getNode()` method onto `Bootstrapper` is not appropriate and violates the design idea.

Reply:
No strong opinion here although I don't agree with the design violation comment. Made the change